components/uaepass-authenticator/src/main/java/org/wso2/carbon/identity/uaepass/authenticator/exception/UAEPassAuthnFailedException.java

```diff
@@ -0,0 +1,18 @@
+package org.wso2.carbon.identity.uaepass.authenticator.exception;
```

Comment:
Add the license header.

Reply:
Fixed with : [bf5488f](https://github.com/wso2-extensions/identity-outbound-auth-uaepass/pull/1/commits/bf5488f695a7cae8933ece11d28fcab07cd8e53f)